config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -57,6 +53,7 @@ presubmits:
         preset-service-account: "true"
       max_concurrency: 8
       name: pull-cert-manager-chart
+      optional: false
```

Comment:
note: the generation code has two types of presubmit; required and optional.

Required presubmits have `always_run = true` and `optional = false`

Optional presubmits have the opposite: `always_run = false` and `optional = true`

`always_run` and `optional` should now be set for all presubmits.